watertap/tools/oli_api/conftest.py

```diff
@@ -88,9 +91,37 @@ def oliapi_instance(
         "config_file": cred_file_path,
     }
     credential_manager = CredentialManager(**credentials, test=True)
-    credential_manager.login()
```

Comment:
Can you clarify why this isn't needed anymore (or why it was there before and is now being removed)?

Reply:
Essentially, `login` is now executed in `__init__` rather than by the user. It's needed to set `refresh_token` and `jwt_token` which are required when using username/password to login